src/app/components/ATIAnalytics/types.ts

```diff
@@ -89,6 +79,13 @@ export interface ATIDataWithContexts {
   atiData: ATIData;
 }
 
+export interface ATIConfigurationDetailsProviders {
+  requestContext: RequestContextProps;
+  serviceContext: ServiceConfig;
+  data?: PageData;
```

Comment:
Rename? (For consistency with EventTracking) 
```suggestion
  pageData?: PageData;
```

Reply:
We think it makes more sense if we retain the prop as `data` for EventTracking for now. If we implement your suggestion currently we'd have to update it for all other page types and their corresponding unit tests for little gain since `pageData` will not be passed in to `ATIAnalytics` down the line (i.e in WSTEAMA-423 - which is coming next). We have instead changed `EventTrackingContextProvider` to use `data` so that it conforms to a pattern